src/components/Card/card.css

```diff
@@ -7,6 +7,12 @@
   border-left-width: 3px;
 }
 
+.decision-card img {
```

Comment:
What does this new CSS class have to do with these changes?

Reply:
It was something I just noticed during testing, that some images on a card render pretty big, so limiting that here. We can make this a separate PR though if needed.

Reply:
No, it's fine - just wanted to make sure this didn't get included by accident